vhdbuilder/packer/components.json

```diff
@@ -199,17 +189,13 @@
       "downloadURL": "mcr.microsoft.com/oss/cilium/operator-generic:*",
       "amd64OnlyVersions": [],
       "multiArchVersions": [
-        "1.13.10",
```

Comment:
same comment here about Cilium, check with Tamilmani and Camryn

Reply:
Have pinged them.